examples/matplotlib/mpl_styles.py

```diff
@@ -20,18 +20,27 @@
     "arviz-whitegrid",
     "arviz-white",
     "arviz-grayscale",
+    ["arviz-white", "arviz-redish"],
+    ["arviz-white", "arviz-bluish"],
+    ["arviz-white", "arviz-orangish"],
+    ["arviz-white", "arviz-brownish"],
+    ["arviz-white", "arviz-purplish"],
+    ["arviz-white", "arviz-cyanish"],
+    ["arviz-white", "arviz-greenish"],
+    ["arviz-white", "arviz-royish"],
+    ["arviz-white", "arviz-viridish"],
+    ["arviz-white", "arviz-plasmish"],
 ]
 
-fig = plt.figure(figsize=(12, 12))
+fig = plt.figure(figsize=(12, 25))
 for idx, style in enumerate(style_list):
-    with az.style.context(style):
-        ax = fig.add_subplot(3, 2, idx + 1, label=idx)
+    with az.style.context(style, after_reset=True):
```

Comment:
What's after reset?

Reply:
reset style settings to the default values and then apply the new style.